libs/remix-ui/workspace/src/lib/actions/workspace.ts

```diff
@@ -68,8 +68,12 @@ export const createWorkspace = async (workspaceName: string, workspaceTemplateNa
     await plugin.workspaceCreated(workspaceName)
 
     if (isGitRepo) {
-      await plugin.call('dGitProvider', 'init', { branch: 'main' })
-      dispatch(setCurrentWorkspaceCurrentBranch('main'))
+      const workspacesPath = plugin.fileProviders.workspace.workspacesPath
+      const allBranches = await getGitRepoBranches(workspacesPath + '/' + workspaceName)
```

Comment:
perhaps we should default to `main` if `allBranches` is empty

Reply:
ok, this is updated